src/core/Akka.TestKit/DelegatingSupervisorStrategy.cs

```diff
@@ -22,15 +22,19 @@ public class DelegatingSupervisorStrategy : SupervisorStrategy
         
         protected override Directive Handle(IActorRef child, Exception exception)
         {
-            var childDelegate = Delegates[child];
-            var handleMethod = typeof(SupervisorStrategy).GetMethod(
-                name: "Handle", 
-                bindingAttr: BindingFlags.Instance | BindingFlags.NonPublic, 
-                binder: Type.DefaultBinder,
-                types: new[] {typeof(IActorRef), typeof(Exception)}, 
-                modifiers: null);
-            var result = (Directive) handleMethod.Invoke(childDelegate, new object[]{ child, exception });
-            return result;
+            if(Delegates.TryGetValue(child, out var childDelegate))
```

Comment:
check if the child `IActorRef` has a strategy specified at all - if not, fall back to the default.